docs/learn-crosschain.md

```diff
@@ -26,7 +30,33 @@ messages.
 - Validators will check a proof that the new candidate for the next parachain block includes the
   processing of the expected ingress messages to that parachain.
 
-## Example
+XCMP queues must be initiated by first opening a channel between two parachains. The channel is
+identified by both the sender and recipeient parachains, meaning that it's a one-way channel. A pair
+of parachains can have at most two channels between them, one for sending messages to the other
+chain and another for receiving messages. Channel will require a deposit in DOT to be opened, which
+will get reciprocated when the channel is closed.
+
+## Horizontal Relay-routed Message Passing (HRMP)
+
+While XCMP is still being implemented, a stop-gap protocol known as HRMP exists in its place. HRMP
```

Comment:
Perhaps here, could you elaborate on what a stop-gap protocol is? And maybe why this specific protocol is needed?